### ISSUE TITLE
Parameter 'showLabels' error when connecting to CAS

### ISSUE THREAD

**Issue by sdimos** [sassoftware/R-swat#29]
Hi! I'm trying to connect to CAS using the following R code running in Rstudio. I have an authfile generated and the required packages are installed.

```
install(swat)
s <- CAS("my.host.com", 443, protocol='https')
```

However, I get this error: 
```
NOTE: Connecting to CAS and generating CAS action functions for loaded
      action sets...
ERROR: Parameter 'showLabels' is not recognized.
ERROR: Expecting one of the following: actionSet, action, dynamicOptions.
ERROR: The action stopped due to errors.
Error in res$results[[1]] : subscript out of bounds
```

Any idea what's causing this error?

**Comment by kesmit13** [comment]
Do you know what version of the server you are trying to connect to?

**Comment by kesmit13** [comment]
This should be fixed in v1.6.2.
